Tests/js/views/dashboard/ez-dashboardblockallcontentview.html

```diff
@@ -0,0 +1,75 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>eZ Dashboard All Content Block View tests</title>
+</head>
+<body>
+<div class="container"></div>
+<div class="outside"></div>
+
+<script type="text/x-handlebars-template" id="dashboardblockallcontentview-ez-template" >
+    <table class="pure-table">
+        <tbody class="ez-allcontent-block-content">
+        {{#each items}}
+            <tr class="ez-allcontent-block-row">
+                <td class="ez-allcontent-block-cell--title">{{ content.name }}</td>
```

Comment:
no need for so many classes, please use minimal template so that the view can be rendered